File: app/src/main/java/mx/dev/franco/automusictagfixer/ui/trackdetail/ValidationWrapper.java
package mx.dev.franco.automusictagfixer.ui.trackdetail;

import androidx.annotation.IntegerRes;

public class ValidationWrapper {

    @IntegerRes
    private int field;
    private int message;

    public ValidationWrapper() {}

    public ValidationWrapper(int field, int message) {
        this();
        this.field = field;
        this.message = message;
    }

    public int getField() {
        return field;
    }

    public void setField(int field) {
        this.field = field;
    }

    public int getMessage() {
        return message;
    }

    public void setMessage(int message) {
        this.message = message;
    }
}

File: app/src/main/java/mx/dev/franco/automusictagfixer/ui/trackdetail/ValidationWrapper.kt
package mx.dev.franco.automusictagfixer.ui.trackdetail

import androidx.annotation.IntegerRes

class ValidationWrapper() {
    @IntegerRes
    var field = 0
    var message = 0

    constructor(field: Int, message: Int) : this() {
        this.field = field
        this.message = message
    }
}